src/test/resources/features/11-allergy/allergies.feature

```diff
@@ -10,9 +10,25 @@ Feature: Allergies Management
     And a user clicks No Known Allergy button
     Then the system add no known allergies into the allergies table
     And close browser
-
+  
   @run
   Scenario: Remove No known allergies
     And a user clicks Remove No Known Allergy icon
     Then the system displays unknown in the allergies table
-    And close browser
\ No newline at end of file
+    And close browser
+    
+  Scenario: Edit allergies
+    And a user clicks Edit Allergy icon
+    Then the system displays the adjustable fields
+    And a user edits the allergies
+    And quit browser
+   
+  Scenario: Delete allergies
+    And a user clicks Delete Allergy icon
+    Then the system prompts the user to confirm the transaction
+    And a user deletes the allergy
+    And quit browser
```

Comment:
Thanks so much for working on this again ,why are we quiting the browser instead of closing, ideally how do cucumber treat these two scenarios, feel free to tell me the difference please thanks

Reply:
> how do cucumber treat these two scenarios, feel free to tell me the difference please thanks

If the Automation process opens only a single browser window, the close() and quit() commands work in the same way. Both will differ in their functionality when there are more than one browser window opened during Automation. close() method  closes the browser window which is currently in focus at that time and the remaining browser windows will not be closed. In contrast, quit() method in turn closes all the browser windows that are open and terminates the WebDriver session

Reply:
Thanks so much for detailed feedback,
This LGTM , however one more thing to go, Can you try to add  these scenarios under edit scenario
 Something like 

Scenario: Edit allergies
    **And a user clicks Edit Allergy icon
    Then the system displays the Edit Allergy Page
    When User clicks Remove Allergy button
    Then the system prompts the user to confirm the transaction
    Then System deletes All Allergyies
    And a user edits the allergies
    And close browser**
 Take a look as you compare it with editing the allergy using this [link  here](https://qa-refapp.openmrs.org/openmrs/allergyui/allergy.page?allergyId=2&patientId=508&returnUrl=%2Fopenmrs%2Fcoreapps%2Fclinicianfacing%2Fpatient.page%3FpatientId%3D508%26&)

Reply:
i think another way is to add 4th and 5th  line under background part cc @k-joseph

Reply:
> Take a look as you compare it with editing the allergy using this [link here](https://qa-refapp.openmrs.org/openmrs/allergyui/allergy.page?allergyId=2&patientId=508&returnUrl=%2Fopenmrs%2Fcoreapps%2Fclinicianfacing%2Fpatient.page%3FpatientId%3D508%26&)

After taking the use case have made appropriate adjustment. Have addressed the idea in the updated PR. Thanks Shariff for the insight

Reply:
> i think another way is to add 4th and 5th line under background part cc @k-joseph

Have not got this point right! Could you bring it home

Reply:
Great did you see this

Reply:
> Great did you see this

What exactly are you pointing at

Reply:
> i think another way is to add 4th and 5th line under background part cc @k-joseph

Which user story are you proposing to add in the background. Are you suggesting these below which have quoted in one of the previous comment.
**_Then the system prompts the user to confirm the transaction
Then System deletes All Allergyies_**